windows/src/desktop/inst/keymandesktop.wxs

```diff
@@ -150,16 +150,16 @@
               </Component>
 
               <Component Id="Reg_RootPath" Guid="*">
-                <RegistryValue KeyPath="yes" Root="HKLM" Key="SOFTWARE\Keyman\Keyman Desktop" Name="root path" Type="string" Value="[INSTALLDIR]" />
-                <RegistryValue Root="HKLM" Key="SOFTWARE\Keyman\Keyman Desktop" Name="version" Type="string" Value="$(var.VERSION)" />
-                <RegistryValue Root="HKLM" Key="SOFTWARE\Keyman\Keyman Desktop" Name="release" Type="string" Value="$(var.RELEASE)" />
-                <RegistryValue Root="HKLM" Key="SOFTWARE\Keyman\Keyman Desktop" Name="charmap source data" Type="string" Value="[INSTALLDIR]" />
-                <RegistryKey ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes" Root="HKLM" Key="Software\Keyman\Keyman Desktop" />
+                <RegistryValue KeyPath="yes" Root="HKLM" Key="SOFTWARE\Keyman\Keyman for Windows" Name="root path" Type="string" Value="[INSTALLDIR]" />
+                <RegistryValue Root="HKLM" Key="SOFTWARE\Keyman\Keyman for Windows" Name="version" Type="string" Value="$(var.VERSION)" />
+                <RegistryValue Root="HKLM" Key="SOFTWARE\Keyman\Keyman for Windows" Name="release" Type="string" Value="$(var.RELEASE)" />
+                <RegistryValue Root="HKLM" Key="SOFTWARE\Keyman\Keyman for Windows" Name="charmap source data" Type="string" Value="[INSTALLDIR]" />
+                <RegistryKey ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes" Root="HKLM" Key="Software\Keyman\Keyman for Windows" />
               </Component>
 
               <Component Id="removeCU" Guid="*">
-                <RegistryValue KeyPath="yes" Root="HKCU" Key="Software\Keyman\Keyman Desktop" Name="Install" Type="string" Value="1" />
-                <RegistryKey ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes" Root="HKCU" Key="Software\Keyman\Keyman Desktop" />
+                <RegistryValue KeyPath="yes" Root="HKCU" Key="Software\Keyman\Keyman for Windows" Name="Install" Type="string" Value="1" />
+                <RegistryKey ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes" Root="HKCU" Key="Software\Keyman\Keyman for Windows" />
```

Comment:
This is a potential breaking change to be aware of. I may want to roll this one back. Thinking about it.

Reply:
I rolled it back.